buildinfo/buildinfo.go

```diff
@@ -17,15 +15,25 @@ var (
 	Version = "n/a"
 )
 
-// Summary prints a summary of all build info.
-func Summary() string {
-	return fmt.Sprintf(
-		"\tversion:\t%s\n\tbuild date:\t%s\n\tgit summary:\t%s\n\tgit branch:\t%s\n\tgit commit:\t%s\n\tgit state:\t%s",
-		Version,
-		BuildDate,
-		GitSummary,
-		GitBranch,
-		GitCommit,
-		GitState,
-	)
+// Summary provides a summary of git information in the binary.
+type Summary struct {
+	GitCommit  string
+	GitBranch  string
+	GitState   string
+	GitSummary string
+	BuildDate  string
+	Version    string
+}
+
+// GetSummary returns a JSON string with a summary of git information.
+func GetSummary() (Summary, error) {
```

Comment:
Maybe I'm missing something or this is a leftover of another logic. Does returning `error` make sense, if the code always returns `nil`?

Reply:
Probably a left over, I'll remove it :+1: